next.config.js

```diff
@@ -25,6 +25,10 @@ const nextConfig = withPWA({
     },
     images: {
         remotePatterns: [
+            {
+                protocol: "https",
+                hostname: "api.rezervo.no",
```

Comment:
Is there a way to make this depend on `NEXT_PUBLIC_CONFIG_HOST`?

Reply:
Jupp. Fixed in https://github.com/mathiazom/rezervo-web/pull/119/commits/e42296305ba6cac806ca7dc845123125cb73cc3d